src/features/Send/useCases/ListSelectedTokens/AddToken/SelectTokenFromListScreen.tsx

```diff
@@ -71,43 +65,37 @@ export const AssetSelectorScreen = () => {
         testID="assetsList"
         estimatedItemSize={78}
       />
-
-      <Actions>
-        <Button
-          outlineOnLight
-          title={strings.sendAllAssets}
-          onPress={() => {
-            allTokensSelected()
-            navigation.navigate('send')
-          }}
-          testID="sendAllAssetsButton"
-        />
-      </Actions>
-    </SafeAreaView>
+    </View>
   )
 }
 
-type SelectableAssetItemProps = AssetItemProps & {
-  onSelect(): void
-}
-const SelectableAssetItem = ({tokenInfo, balance, onSelect}: SelectableAssetItemProps) => {
+type SelectableAssetItemProps = Omit<AssetItemProps, 'quantity'>
+const SelectableAssetItem = ({tokenInfo}: SelectableAssetItemProps) => {
+  const {tokenSelectedChanged, amountChanged} = useSend()
+  const {spendable} = useTokenQuantities(tokenInfo.id)
+  const navigation = useNavigation<TxHistoryRouteNavigation>()
+
+  const onSelect = () => {
+    tokenSelectedChanged(tokenInfo.id)
+
+    // if the token is indivisible, we don't need to ask for the amount
+    if (Quantities.isIndivisible(spendable, tokenInfo.decimals)) {
```

Comment:
what is `indivisible`?

Reply:
minimal amount, looking for a better name 6 dec = .000001, 0 dec = 1.